package.json

```diff
@@ -111,10 +111,10 @@
     "@bbc/psammead-useful-links": "3.0.14",
     "@bbc/psammead-visually-hidden-text": "2.0.3",
     "@bbc/web-vitals": "1.0.2",
-    "@emotion/cache": "11.1.3",
-    "@emotion/react": "11.1.4",
+    "@emotion/cache": "11.0.0",
```

Comment:
Shouldn't this be at the latest version? Probably should test this with the latest version

Reply:
I must have missed this when upgrading the others 🙁